index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Portofolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class= "container">
+        <div class="header">
+            <div><span>Abdi Bekele</span></div>
+            <div class="humber-button"><i class="fa fa-bars"></i></div>
+        </div>
```

Comment:
- [ ] I suggest you use a `header` tag in this part of your code. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

Reply:
- [ ] Consider wrapping the `nav` tag around the hamburger menu. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.

Reply:
- [ ] I suggest you use an <a> tag for the logo. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.